src/openrct2/ride/RideConstruction.cpp

```diff
@@ -1200,42 +1202,39 @@ money32 set_operating_setting_nested(ride_id_t rideId, RideSetSetting setting, u
     return res->Error == GameActions::Status::Ok ? 0 : MONEY32_UNDEFINED;
 }
 
-static int32_t loc_6CD18E(
+// for a 3xN or nx3 grid, find the direction that faces the inside.
```

Comment:
So this does not work for, say 4×4?

Reply:
Flat ride path never hits this. For an NxN flat ride base, this function has to be called with N+2xN+2 grid.